lms/templates/ace_common/edx_ace/common/base_body.html

```diff
@@ -62,8 +62,8 @@
                     <tr>
                         <td width="70">
                             <a href="{% with_link_tracking homepage_url %}"><img
-                                    src="https://courses.xpro.mit.edu/static/mitxpro-theme/images/logo.png" width="130"
-                                    height="40" alt="{% filter force_escape %}{% blocktrans %}Go to {{ platform_name }} Home Page{% endblocktrans %}{% endfilter %}"/></a>
+                                    src="https://courses-rc.xpro.mit.edu/static/mitxpro-theme/images/logo.svg" width="125"
```

Comment:
@asadali145 I've changed it to use the RC instance for the logo. We will need this for testing in the RC instance because we won't have the new image on prod until we have released it. Once we have tested it I can revert it to using the prod instance.

Reply:
Why don't we use the configs as edX does for the email logo?
https://github.com/openedx/edx-platform/blob/41ee16a6b74055689fc1d7a3c591275af4ff42c8/openedx/core/djangoapps/ace_common/templates/ace_common/edx_ace/common/base_body.html#L82
https://github.com/openedx/edx-platform/blob/41ee16a6b74055689fc1d7a3c591275af4ff42c8/openedx/core/djangoapps/ace_common/template_context.py#L47
https://github.com/openedx/edx-platform/blob/03b189fc7c31b3e626c9a199cd9083fe514509bf/lms/djangoapps/branding/api.py#L648

Reply:
Thanks for referencing this. I already tried that before I made this change, That `logo_url` is not part of the context at least in our theme. The reasons I discarded the thought of using `logo_url` tag:
1. I tried to debug it but I wasn't getting the URL at all
2. That tag uses [get_logo_url_for_email](https://github.com/openedx/edx-platform/blob/master/lms/djangoapps/branding/api.py#L648) Which will require us to add new config variables
3. I think we should do a rebase of the theme, or at least pull in the latest email_base.html in our theme but at some later point.

I also plan to create a ticket for this specific task. Let me know what you think?

Reply:
Sounds good to me.